File: C/05-Operadores Aritméticos/Precedencia.c
#include<stdio.h>

int main(void)
{
    //A precedencia dos operadores aritméticos em C é
    //()
    //*/%
    //+-
    //Mesma precedencia é feito da esquerda para direita
    int x = 7;
    int y = 9;
    //parenteses tem maior precedência
    int media = (x + y) / 2;
    //multiplicação tem maior precedência que soma
    int valor = x + y * y;//7 + 81
    //mesma precedencia é resolvido da esquerda para direita
    int valor2 = x + y - x;

    printf("%d \n",media);
    printf("%d \n",valor);
    printf("%d \n",valor2);
    return 0;
}

File: PHP/03-Operadores Aritméticos/Precedencia.php
<?php

//Alguns operadores são sempre calculados antes de outros

//No php a precedência dos operadores é
//1- * / %
//2- + -
$n1 = 2 + 5 * 4; // 2 + 20
$n2 = 3 - 4 / 2; // 3 - 2 
$n3 = 1 + 3 % 2; // 1 + 1

//Se quisermos forçar uma parte da expressão a ser calculada
//primeiro usamos (). Tudo o que tiver entre parenteses será
//calculado primeiro

$media = (1 + 2 + 3) / 3; // 6 / 3 

//Operadores de mesma precedencia são calculados da esq para dir
$n4 = 1 + 2 - 3; // 3 - 3

echo $n1;
echo'<br/>';
echo $n2;
echo'<br/>';
echo $n3;
echo'<br/>';
echo $media;
echo'<br/>';
echo $n4;
echo'<br/>';

?>